string chess_code
---
geometry_msgs/TransformStamped stampedpose
